package.json

```diff
@@ -34,7 +34,7 @@
     "zx": "^7.2.3"
   },
   "scripts": {
-    "build": "npm run build:assets && npm run archive",
```

Comment:
It seems like we are going to replace the archive command? If this is the case, maybe it makes sense to remove the `archive.exclude` arguments from composer.json? Keep in mind although it is called exclude we do a `!` so they are actually includes.

We should also remove the old archive command

Reply:
Done in 9215cf69c667681fe97648d9e1448b07f6791b1e